storage_visualization/submit.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+"""Submits `main.py` to the analysis-runner, with a full set of dataset names based on a yaml config."""
+
+import datetime
+import subprocess
+import os
+import sys
+import yaml
+
+ALL_DATASETS = 'all-datasets'
+
+
+def main():
+    """Main entrypoint."""
+    if len(sys.argv) != 2:
+        print('Usage: submit.py <config.yaml>')
+        sys.exit(1)
+
+    with open(sys.argv[1], encoding='utf-8') as f:
+        config = yaml.safe_load(f)
+
+    # Generate a list of all datasets to invoke main.py with.
+    datasets = sorted(list(set(config.keys()) - {ALL_DATASETS}))
+
+    subprocess.check_output(
```

Comment:
Is the analysis-runner dependency installed? If so, would be cleaner to call the CLI method directly, https://github.com/populationgenomics/analysis-runner/blob/c4f8f91c6fb0664923d29d11ea356865df0001ea/analysis_runner/cli_analysisrunner.py#L98